LSP/Example1.md

```diff
@@ -0,0 +1,23 @@
+### Usage
+
+---
+#### Where
+
+https://github.com/tradingview/lightweight-charts/blob/master/src/formatters/percentage-formatter.ts#L3-L11
+
+#### Why
+
+class `PercentageFormatter` - we can safely swap usage of `PriceFormatter` with `PercentageFormatter`. The code will not break since the contract is respected.
+
+------------------------
+
+### Violation
+
+---
+#### Where
+
+https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Engines/engine.ts
```

Comment:
➕ ➖
I believe there is not only LSP (resize method is example), but also SRP, OCP, DIP violations.

Reply:
How `resize` method violates LSP?

Reply:
👎  SRP violation